### PR TITLE
#ibranista-learning

### PR DESCRIPTION
**Pull request by Ibranista**
#1. fixed linter issues
#2. fixed the rule-not-found problem for tailwindCSS
#3. fixed footer background-color 
#4. fixed main program section's card background-color to make it look like the original design
#5. Corrected the footer alignment on both desktop and small screen devices.
#6. fixed the background color of meetings section in side about page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,16 +69,6 @@ To use this repository you need to have basic skills in the following languages
...
-
-## 🤝 Contributing <a name="contributing"></a>
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](https://github.com/).
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- SUPPORT -->
```
Comment (cwaku) [inline README.md:73]: - [ ] I recommend that the README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @Ibranista 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️, but you are almost there! 🏃🏾 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 

## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance to learn practical coding skills and new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them to get your project approved ✅

- [ ] Great job with the video presentation but you happen to  have missed one of the requirements of the project documentation.The video should contain a highlight about a piece of the code the you find interesting or are proud of making. kindly remake the video and add this piece to pass the acceptance criteria.

- [ ] Following the design guidelines is very important for this project and you have done a great job so far but you still missing some, kindly follow the following recommendations to do an even better job.
##### Footer
- [ ] It should contain a logo aligned to the left and text aligned to the right. Your current has them stacked on top of each other on the mobile version.
  
Your Design | Design Template
--|--
image | image
   
###### Mobile Size

- [ ] It should have a white background. Kindly fix that.

Your Design | Design Template
--|--
image | image

###### Desktop Size

- [ ] About Page: It should have a dark background.
- [ ] Home Page: It should have a white background. 

#### Design and Styling

- [ ] The pages should use the exact same colors, for text, backgrounds, borders, etc, from the original design. Kindly update your colors to the colors provided below.

* Basic color swatch: #272A31, #EC5242, #D3D3D3

Your Design | Design Template
--|--
image | image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -86,16 +69,6 @@ To use this repository you need to have basic skills in the following languages
...
```
Comment (Nemwel-Boniface) [inline README.md:72]: - [ ] As requested by the [previous reviewer](https://github.com/Ibranista/ibranista-university/pull/3#discussion_r1031828404), kindly give credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Ibranista    :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job addressing some of the changes that were requested by a previous reviewer :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [ ] As you have been requested by the previous reviewer, and in the project requirements, you are required to follow the design in regard to typography, colors, font sizes etc.

**These are the sections as requested by the previous reviewer that needs changes:**

image

image

**Kindly use the correct colors/ background colors as they are in the project design**

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @Ibranista   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**